README.md

```diff
@@ -1,45 +1,33 @@
 # script-runner
 
-[About this repo](#about-this-repo) |  [Commands](#commands-optional) | [How to use this repo](#how-to-use-this-repo)
+[About this repo](#about-this-repo) | [Development](#development) | [Usage](#usage)
 
 ## About this repo
+
 **Introduction:**
 
-This is a simple CLI (`script-runner`) which allows you to run Wasm
-scripts intended for ShopifyVM. Scripts will run using
+This is a simple CLI (`script-runner`) which allows you to run Shopify Functions intended for ShopifyVM. Functions will run using
 the provided JSON input file and their output will be printed as JSON
 upon completion.
 
-By default, the script is expected to be named `script.wasm` in the
+By default, the function is expected to be named `script.wasm` in the
```

Comment:
The script is expected to be located in the current directory and named `script.wasm`. Use the `-s` option to change this default.

Reply:
But aren't we referring to this as a "function"? We should also update the default path to `function.wasm`, shouldn't we?